internal/pkg/generate/generate_scripts.go

```diff
@@ -580,8 +580,11 @@ func joinIfNotEmpty(strs ...string) string {
 }
 
 // imageOrEmpty returns the image of the client if it is not nil, otherwise returns an empty string
-func imageOrEmpty(cls *clients.Client) string {
+func imageOrEmpty(cls *clients.Client, latest bool) string {
 	if cls != nil {
+		if latest {
+			return "latest"
+		}
```

Comment:
I have the feeling you are setting `latest` as the entire image name instead as a tag and the tests are missing this.